harness/tests/test/unit/test_rpc.gd

```diff
@@ -0,0 +1,25 @@
+extends "res://addons/gut/test.gd"
+
+func test_nullables_are_correctly_set_from_function_lika_a_ready_call():
```

Comment:
how will RPC methods work when called in Kotlin?

Reply:
https://github.com/utopia-rise/godot-kotlin-jvm/blob/a486f8b5e89c31ae89123685a2167ef6820d1776/harness/tests/src/main/kotlin/godot/tests/rpctests/RpcTests.kt#L26
Or do you mean something different?

Reply:
Ahh, I'm blind lol.